src/Hangfire.Tags.PostgreSql/PostgreSqlTagsServiceStorage.cs

```diff
@@ -56,7 +57,7 @@ public IEnumerable<string> SearchTags(string tag, string setKey)
             return monitoringApi.UseConnection(connection =>
             {
                 var sql =
-                    $@"select Value from {_options.SchemaName}.Set s where s.Key like @setKey + ':%' + @tag + '%'";
+                    $@"select Value from {_options.SchemaName}.Set s where s.Key like (@setKey || ':%' || @tag || '%')";
```

Comment:
Should this be a separate commit since it fixes a different issue than the connection exhaustion? (Can still be this PR)

Reply:
yup